src/NUnitConsole/nunit-console/ConsoleRunner.cs

```diff
@@ -221,15 +221,17 @@ private void RedirectOutputAsRequested()
         {
             if (_options.OutFile != null)
             {
-                var outStreamWriter = new StreamWriter(Path.Combine(_workDirectory, _options.OutFile));
-                outStreamWriter.AutoFlush = true;
+                var outStreamWriter = new StreamWriter(Path.Combine(_workDirectory, _options.OutFile)) {
+                    AutoFlush = true
+                };
```

Comment:
This construct works in all the platforms for which we currently build ConsoleRunner. It may not work on all platforms to which we port it in the future. That's why we make so little use of this. Also... not in the standard. :-)